war/src/main/js/pluginSetupWizardGui.js

```diff
@@ -547,13 +572,14 @@ var createPluginSetupWizard = function(appendTarget) {
 					}
 				}
 
-				$c = $('.install-console-scroll');
-				if($c.is(':visible')) {
+				var $c = $('.install-console-scroll');
+				if($c && $c.is(':visible') && !$c.data('userScrolled')) {
+					$c.data('wasAutoScrolled', true);
 					$c.scrollTop($c[0].scrollHeight);
 				}
 
 				// keep polling while install is running
-				if(complete < total || data.state === 'INITIAL_PLUGINS_INSTALLING') {
+				if(complete < total && data.state === 'INITIAL_PLUGINS_INSTALLING') {
```

Comment:
this was what was causing the 'hang'... everything done but since there were failures it didn't transition state